opba-protocols/xs2a-protocol-tests/xs2a-bdd-sandbox/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/sandbox/servers/SandboxServers.java

```diff
@@ -55,4 +66,53 @@ public SELF enabled_redirect_sandbox_mode(String aspspProfileUri) {
 
         return self();
     }
+
+    public SELF create_new_user(String tppServerUrl, String login, String password) {
+        String body = readResource("restrecord/tpp-ui-input/params/new-user-registration-body.json")
+                              .replace(USER_NAME_PLACEHOLDER, login)
+                              .replace(PASSWORD_PLACEHOLDER, password)
+                              .replace(EMAIL_PLACEHOLDER, login + "@gmail.com");
+
+        RestAssured
+                .given()
+                    .contentType(MediaType.APPLICATION_JSON_VALUE)
+                    .body(body)
+                .when()
+                    .post(tppServerUrl + "/tpp/users")
+                .then()
+                    .statusCode(HttpStatus.CREATED.value());
+
+        return self();
+    }
+
+    public SELF create_account_for_user(String tppServerUrl, String user) {
+        String body = readResource("restrecord/tpp-ui-input/params/new-user-new-account-registration.json")
+                .replace(ACCOUNT_ID_PLACEHOLDER, ACCOUNT_ID);
+
+        RestAssured
+                .given()
+                    .contentType(MediaType.APPLICATION_JSON_VALUE)
+                    .body(body)
+                .when()
+                    .post(tppServerUrl + "/tpp/accounts")
+                .then()
+                    .statusCode(HttpStatus.CREATED.value());
+
+        return self();
+    }
+
+    public SELF deposit_user_account(String tppServerUrl, String user) {
```

Comment:
Please rename `deposit_user_account` -> `deposit_to_sandbox_tpp_management_user_account`
Please rename `tppServerUrl` to sandboxTppManagementServerUrl

TPP is actually OpenBanking, so it is confusing

Reply:
renamed